{
  "comments": [
    {
      "key": {
        "uuid": "32d111fc_1ee27e3d",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 2
      },
      "lineNbr": 0,
      "author": {
        "id": 1019050
      },
      "writtenOn": "2020-09-24T19:39:35Z",
      "side": 1,
      "message": "libbacktrace has some failures, but I think they existed before. But I\u0027ll try to fix them or disable the tests.",
      "revId": "482d9a5ff9e3cd6252484cd9b894cccb7a82a19d",
      "serverId": "85c56323-6fa9-3386-8a01-6480fb634889",
      "unresolved": false
    }
  ]
}